examples/vue/.gitignore

```diff
@@ -0,0 +1,28 @@
+# Logs
```

Comment:
I decided to keep this and a couple of other files since they were added by the Vue project generator. They will help our users if they decide to copy the entire folder and use it as a starter project.